pkg/config/config.go

```diff
@@ -647,6 +647,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("kubernetes_apiserver_tls_verify", true)
 	config.BindEnvAndSetDefault("leader_lease_duration", "60")
 	config.BindEnvAndSetDefault("leader_election", false)
+	config.BindEnvAndSetDefault("lease_name", "datadog-leader-election")
```

Comment:
Would it make sense to prefix this key with something to indicate it is related to leader election? Something like `leader_lease_name` or `cluster_agent.lease_name` perhaps.